pages/index.tsx

```diff
@@ -20,6 +20,15 @@ export default function Index() {
       >
           Send a new Email
       </Button></Link><br />
+      <Link href='/lostfound'><Button
```

Comment:
link here will not work like this, but not very important right now, ok for temporary thing